src/__tests__/components/CountriesContainer.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { BrowserRouter } from 'react-router-dom';
+import renderer from 'react-test-renderer';
+import store from '../../__mocks__/configureStore';
+import CountriesContainer from '../../components/CountriesContainer';
+
+describe('Countries snapshot', () => {
+  test('renders correctly', () => {
+    const tree = renderer
+      .create(
+        <Provider store={store}>
+          <BrowserRouter>
+            <CountriesContainer />
+          </BrowserRouter>
+        </Provider>,
+      )
+      .toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] Following the project requirements, You should also use React Testing Library to simulate user interaction for some components. Check out this [link ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/articles/testing_react_with_react_testing_library.md)to learn how to do that.